sdk/monitor/Azure.ResourceManager.Monitor/src/autorest.md

```diff
@@ -260,6 +260,7 @@ rename-mapping:
   TimeSeriesElement: MonitorTimeSeriesElement
   StorageBlobDestination.storageAccountResourceId: -|arm-id
   StorageTableDestination.storageAccountResourceId: -|arm-id
+  PrivateLinkScopedResource: DataCollectionRulePrivateLinkScopedResource
```

Comment:
Could we change the name to `DataCollectionRulePrivateLinkScopedResourceInfo` since Resource is a reserved suffix in mgmt plane SDKs?